src/dbt/kipptaf/models/performance_management/intermediate/int_performance_management__observation_details.sql

```diff
@@ -113,6 +110,9 @@ with
             {{ ref("stg_performance_management__observation_details") }} as od
             on m.observation_id = od.observation_id
             and m.score_measurement_id = od.score_measurement_id
+        left join
+            {{ ref("base_people__staff_roster_history") }} as srh
```

Comment:
this is long by date, did you mean to join to `base_people__staff_roster`?

Reply:
just kidding